inlong-dashboard/src/components/AccessHelper/DataStorageEditor/DetailModal.tsx

```diff
@@ -186,11 +187,27 @@ const Comp: React.FC<DetailModalProps> = ({
       {
         name: 'sinkName',
         type: 'input',
-        label: t('components.AccessHelper.DataStorageEditor.SinkName'),
-        rules: [{ required: true }],
+        label: t('components.AccessHelper.StorageMetaData.SinkName'),
+        rules: [
+          { required: true },
+          {
+            pattern: /^[a-zA-Z][a-zA-Z0-9_]*$/,
+            message: i18n.t('components.AccessHelper.StorageMetaData.SinkNameRule'),
+          },
+        ],
         props: {
           disabled: !!id,
         },
+      },
+      {
+        name: 'description',
+        type: 'textarea',
+        label: i18n.t('components.AccessHelper.StorageMetaData.Description'),
+        rules: [{ required: false }],
```

Comment:
This is not required, no need to set rules.

Reply:
OK, removed.